changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+author: "Sosuke Suzuki"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.4: new bracketSameLine option and TypeScript 4.4 support!"
+---
+
+This release renames the `jsxBracketSameLine` option to `bracketSameLine`, which supports HTML, Vue, and Angular in addition to JSX. The old name has been deprecated.
+
+We’ve also added support for [TypeScript 4.4](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html), including new syntax features such as [class `static` blocks](https://github.com/tc39/proposal-class-static-block).
+
+If you enjoy Prettier and would like to support our work, consider supporting us directly via [our OpenCollective](https://opencollective.com/prettier) or by supporting the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel).
```

Comment:
After looking at this again the repetition of “support” doesn’t sound right, oops!

```suggestion
If you enjoy Prettier and would like to support our work, consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel).
```